pennylane_lightning/core/src/simulators/lightning_gpu/utils/MPI_helpers.hpp

```diff
@@ -0,0 +1,150 @@
+#pragma once
```

Comment:
```suggestion
// Copyright 2022-2023 Xanadu Quantum Technologies Inc.

// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at

//     http://www.apache.org/licenses/LICENSE-2.0

// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
#pragma once
```

Reply:
Nice catch, thanks!

Reply:
Was this added, as I cannot see the copyright notice?

Reply:
Done!